lib/token_util.py

```diff
@@ -85,7 +85,8 @@ def token_str_expired(token_str):
     expired = True
     try:
         decoded = jwt.decode(
-            token_str, options={"verify_signature": False, "verify_aud": False, "verify_exp": True, "verify_nbf": True}
+            token_str.strip(),
```

Comment:
Use `token_str.strip("\n ")` to protect also against spaces like it was before

Reply:
Bruno assured that this is the default behavior, space and newlines are both stripped